motion/ruby_motion_query/utils.rb

```diff
@@ -118,6 +118,22 @@ def project_path
 
           ppath
         end
+
+        # To use this, put this in your Rakefile:
+        #   app.development do
+        #     app.info_plist["ProjectBuildTime"] = Time.now
+        #   end
+        def build_time
+          if btime = NSBundle.mainBundle.infoDictionary["ProjectBuildTime"]
```

Comment:
it's more idiomatic to put `unless btime = NSBundle.mainBundle.infoDictionary["ProjectBuildTime"]` and have no else statement.